research/flamby/fed_isic2019/fenda_mkmmd/run_fold_experiment.slrm

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=8
+#SBATCH --gres=gpu:1
+#SBATCH --mem=32G
+#SBATCH --partition=a40
+#SBATCH --qos=m
+#SBATCH --job-name=fl_five_fold_exp
+#SBATCH --output=%j_%x.out
+#SBATCH --error=%j_%x.err
+#SBATCH --time=12:00:00
+
+###############################################
+# Usage:
+#
+# sbatch research/flamby/fed_isic2019/fenda_mkmmd/run_fold_experiment.slrm \
+#   path_to_config.yaml \
+#   path_to_folder_for_artifacts/ \
+#   path_to_folder_for_dataset/ \
+#   path_to_desired_venv/ \
+#   client_side_learning_rate_value \
+#   server_address
+#
+# Example:
+# sbatch research/flamby/fed_isic2019/fenda_mkmmd/run_fold_experiment.slrm \
+#   research/flamby/fed_isic2019/fenda_mkmmd/config.yaml \
+#   research/flamby/fed_isic2019/fenda/hp_results/ \
+#   /Users/david/Desktop/FLambyDatasets/fedisic2019/ \
+#   /h/demerson/vector_repositories/fl4health_env/ \
+#   0.0001 \
```

Comment:
This example is leaving off mu and gamma values as part of the args I think?

Reply:
Good catch